libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -732,6 +732,14 @@ AP_InertialSensor::init(uint16_t sample_rate)
         _gyro_harmonic_notch_filter[i].init(_gyro_raw_sample_rates[i], _calculated_harmonic_notch_freq_hz,
              _harmonic_notch_filter.bandwidth_hz(), _harmonic_notch_filter.attenuation_dB());
     }
+
+    for (uint8_t i=0; i<get_accel_count(); i++) {
+    	_accel_harmonic_notch_filter[i].allocate_filters(_harmonic_notch_filter.harmonics(),
```

Comment:
I think by default we should allocate a single notch for accels and probably ignore this config option

Reply:
Ok, I agree it will free up some cpu, but how can we choose a single frequency?
(looking forward to your #14470)